### DISCUSSION TITLE
Why Devs changed the UI

### DISCUSSION THREAD

**Discussion by anik2k06** [wavetermdev/waveterm#980] [General]
I think the old UI is more intuitive and easy to use then why this change?

**Comment by intaek-h** [comment]
me too. I abandoned Wave terminal after the release.

**Comment by xxnuo** [comment]
Perhaps we could propose or fork the legacy version; I feel like many people don't care about a multi-functional terminal, they just want a user-friendly professional one.

Warp is a mark, focusing on the terminal experience, with no need for customization, just like Apple products, Good looking and practical, not the dark, programmer-nerd style window.

Wave could have been a better open-source version of Warp, too bad it didn't happen.

**Comment by Bengerthelorf** [comment]
I don't understand either. After the update, it discarded the independent input box, which is crucial when needing to connect via SSH under poor network conditions—at least for me. It lacks the convenient features that were already well-implemented before... After spending several hours trying to restore my original workflow to no avail, I quit. As it stands, it has essentially become a GUI version of tmux with AI integration, indistinguishable from the rest.
